File: Modul 3/Soal 1/PRAK301-2410817220018-ANGGRAENI DWI ZAHRA.py
a = input ("Masukkan Nilai a = ")
b = input ("Masukkan Nilai b = ")
c = input ("Masukkan Nilai c = ")

if a > b > c:
    print (c, a, b)
elif a > c > b:
    print (b, c, a)
elif b > a > c:
    print (c, a, b)
elif c > a > b:
    print (b, a, c)
elif b > c > a:
    print (a, c, b)
elif c > b > a:
    print (a, b, c)

File: Modul 3/Soal 1/PRAK301-2410817220018-ANGGRAENI-DWI-ZAHRA.c
#include <stdio.h>
int main() { 
    int a, b, c;
    printf("Masukkan Bilangan a: ");
    scanf("%d", &a);
    printf("Masukkan Bilangan b: ");
    scanf("%d", &b);
    printf("Masukkan Bilangan c: ");
    scanf("%d", &c);

    if (a <= b && a <= c) {
        if ( b <= c ) {
        printf("%d %d %d", a, b, c);    
        }  else {
        printf("%d %d %d", a, c, b);
        }
    } else if (b <= a && b <= c) {
        if (a <= c) {
        printf("%d %d %d", b, a, c);
        } else {
            printf("%d %d %d", b, c, a);
        }
    } else {
        if (a <= b) {
        printf("%d %d %d", c, a, b);
        }
        else if (b <= a) {
        printf("%d %d %d", c, b, a);
        }
    }
    return 0;
}